### PR TITLE
feat: update login page UI components and remove unnecessary code in home page

### PR DESCRIPTION
**Pull request by sidarth-23**
The login page UI components have been updated to match the latest design. The changes include:
- Added AuthPage component from "@repo/ui/components"
- Updated the layout and styling of the login form
- Removed unnecessary imports and code in the Home page component

## Summary

- [x] Tested (Must)
- [ ] Test Case added
- [x] Build Successful (Must)
- [x] Sufficient Code comments added (Must)
- [ ] Attached Screenshots / Videos 
- [x] All Relevant Documents added

### CODE REVIEW

File: apps/admin/app/layout.tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ export default function RootLayout({
 }>) {
   return (
     <html lang="en">
-      <body>
+      <body className="h-screen w-full">
```
Comment (yogai53) [inline apps/admin/app/layout.tsx:16]: import {Body} from "@repo/ui/xyz"
```suggestion
      <Body className="h-screen w-full">
```

File: apps/admin/app/login/page.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      <AuthPage.InputsWrapper>
+        <AuthPage.InputField
+          fieldName="Email"
+          type="email"
+          placeholder="Email"
+        />
+        <AuthPage.InputField
+          fieldName="Password"
+          type="password"
+          placeholder="Password" />
```
Comment (yogai53) [inline apps/admin/app/login/page.tsx:19]: Pls format it.

**Review by yogai53 — APPROVED** [review]

File: apps/admin/app/login/page.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import { AuthPage } from "@repo/ui/components";
+
+export default function LoginPage() {
+  return (
+    <AuthPage.PageWrapper
+      pageTitle="Login"
+      cardTitle="Login"
+      cardDescription="Enter you details to get it"
```
Comment (yaju09) [inline apps/admin/app/login/page.tsx:8]:
```suggestion
      cardDescription="Enter your details to get it"
```

**Review by yaju09 — CHANGES_REQUESTED** [review]

**Review by elizabeththomas92 — APPROVED** [review]

**Comment by sidarth-23** [comment]
Closing this as we changed the course of action of development

**Pull request closed without merging**